README.md

```diff
@@ -111,11 +111,10 @@ cmake -DCMAKE_CXX_COMPILER=g++ \
 This way, all the functionalities are executed on A100 GPUs.
```

Comment:
I think it's not the best idea to duplicate install instructions on both the readme and on the doc. This is an error-prone process (DRY). Since GitHub can display RST files, a different approach is possible.

You should have a condensed section on how to use KokkosFFT as a subdirectory dependency (this is likely how a programmer would start using this library), then have an anchor to the documentation where the complete install process is given:

```md
Further installation details are provided in the [documentation](./docs/intro/building.rst).
```

I think this approach could be used for other sections of the readme. In other words, this file (which is the first contact for a potential user) would be a simple starter, redirecting to more complete documentation pages.

Reply:
It is indeed a duplicate if people read both `README.md` and docs. However, people may not necessarily read the docs. Not sure what is the right way.